src/main/java/Dragons/Dragons.java

```diff
@@ -0,0 +1,48 @@
+package Dragons;
+
+public class Dragons {
+    private String name;
+    private int age;
+    private int weigth;
+    private  typeD type;
+
+
+
+    public Dragons (String name, int age, int weigth, typeD type){}
```

Comment:
constructor doesn't have any logic in the body. Did you mean no args constructor?